In [38]:
import pandas as pd

df = pd.read_csv("KAG_conversion_data.csv")

type(df)

df.head(10)


ad_id  xyz_campaign_id  ...  Total_Conversion Approved_Conversion
0  708746              916  ...                 2                   1
1  708749              916  ...                 2                   0
2  708771              916  ...                 1                   0
3  708815              916  ...                 1                   0
4  708818              916  ...                 1                   1
5  708820              916  ...                 1                   1
6  708889              916  ...                 1                   0
7  708895              916  ...                 1                   1
8  708953              916  ...                 1                   0
9  708958              916  ...                 1                   0

[10 rows x 11 columns]

In [7]:
df.columns

Index(['ad_id', 'xyz_campaign_id', 'fb_campaign_id', 'age', 'gender',
       'interest', 'Impressions', 'Clicks', 'Spent', 'Total_Conversion',
       'Approved_Conversion'],
      dtype='object')

In [8]:
df.index

RangeIndex(start=0, stop=1143, step=1)

In [9]:
df.shape

(1143, 11)

In [13]:
linhas,colunas= df.shape

In [21]:
print(f"O numero de linhas e = {linhas}")

O numero de linhas e = 1143


In [22]:
linhas

1143

In [23]:
colunas

11

In [25]:
type(df)

pandas.core.frame.DataFrame

In [30]:
df ["Clicks"]

0         1
1         2
2         0
3         1
4         1
       ... 
1138    252
1139    120
1140     28
1141    135
1142    114
Name: Clicks, Length: 1143, dtype: int64

In [28]:
df['interest']

0        15
1        16
2        20
3        28
4        28
       ... 
1138    109
1139    110
1140    111
1141    113
1142    114
Name: interest, Length: 1143, dtype: int64

In [32]:
df ["xyz_campaign_id"]

0        916
1        916
2        916
3        916
4        916
        ... 
1138    1178
1139    1178
1140    1178
1141    1178
1142    1178
Name: xyz_campaign_id, Length: 1143, dtype: int64

In [33]:
ser_interest = df["interest"]

In [34]:
type(ser_interest)

pandas.core.series.Series

In [35]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1143 entries, 0 to 1142
Data columns (total 11 columns):
 #   Column               Non-Null Count  Dtype  
---  ------               --------------  -----  
 0   ad_id                1143 non-null   int64  
 1   xyz_campaign_id      1143 non-null   int64  
 2   fb_campaign_id       1143 non-null   int64  
 3   age                  1143 non-null   object 
 4   gender               1143 non-null   object 
 5   interest             1143 non-null   int64  
 6   Impressions          1143 non-null   int64  
 7   Clicks               1143 non-null   int64  
 8   Spent                1143 non-null   float64
 9   Total_Conversion     1143 non-null   int64  
 10  Approved_Conversion  1143 non-null   int64  
dtypes: float64(1), int64(8), object(2)
memory usage: 98.4+ KB


Dataframe. Não há dados faltantes (todos aparecem com 1143), apresenta os tipos numéricos. Há 1 coluna do tipo float, 8 colunas do tipo inteiro e 2 colunas do tipo objeto

In [47]:
df. tail

<bound method NDFrame.tail of         ad_id  xyz_campaign_id  ...  Total_Conversion Approved_Conversion
0      708746              916  ...                 2                   1
1      708749              916  ...                 2                   0
2      708771              916  ...                 1                   0
3      708815              916  ...                 1                   0
4      708818              916  ...                 1                   1
...       ...              ...  ...               ...                 ...
1138  1314410             1178  ...                13                   2
1139  1314411             1178  ...                 3                   0
1140  1314412             1178  ...                 2                   0
1141  1314414             1178  ...                 8                   2
1142  1314415             1178  ...                 5                   2

[1143 rows x 11 columns]>